code/game/machinery/telecomms/presets.dm

```diff
@@ -66,7 +66,7 @@
 
 /obj/machinery/telecomms/receiver/preset_right/initialize()
 	..()
-	freq_listening = list(RESPONSE_FREQ, COMM_FREQ, ENG_FREQ, SEC_FREQ, COMMON_FREQ) //ert, command, engineering, security
+	freq_listening = list(RESPONSE_FREQ, COMM_FREQ, ENG_FREQ, SEC_FREQ, COMMON_FREQ, AIPRIV_FREQ) //ert, command, engineering, security
```

Comment:
I like how the useless comment is now also outdated

Reply:
why read comments when you can read code, amirite